robinhood.d.ts

```diff
@@ -0,0 +1,265 @@
+import request = require('request')
+
+declare function robinhood(options: robinhood.Options.WebApiOpts, callback: robinhood.InitCallback): robinhood.RobinhoodWebApi;
+
+declare namespace robinhood {
+
+  export namespace Options {
+    interface InitResponse {
+      mfa_required: boolean
+    }
+
+    interface WebApiOpts {
+      username?: string
+      password?: string
+      token?: string
+    }
+
+    interface OrdersOptions {
+      updated_at: string
+      instrument: string
+    }
+
+    interface EarningsOptionsWithSymbol {
+      range?: number
+      symbol: string
+    }
+
+    interface EarningsOptionsWithInstrument {
+      range?: number
+      instrument: string
+    }
+
+    interface BuySellOptions {
+      type: OrderType
+      quantity: number
+      bid_price: number
+      instrument: {
+        url?: string
+        symbol?: string
+      }
+      trigger: TriggerType
+      time: TimeInForceType
+    }
+  }
+
+  type TagType = TagTypes
+  type OrderType = OrderTypes
+  type IntervalType = IntervalTypes
+  type SpanType = SpanTypes
+  type TriggerType = TriggerTypes
+  type TimeInForceType = TimeInForceTypes
+  type InitCallback = (data?: Options.InitResponse) => void
+
+  enum TagTypes {
+    TopTen = '10-most-popular',
+    TopOneHundred = '100-most-popular'
+  }
+
+  enum TimeInForceTypes {
+    Immediate = 'immediate',
+    Day = 'day'
+  }
+
+  enum TriggerTypes {
+    GoodForDay = 'gfd',
+    GoodTillCancelled = 'gtc',
+    OrderCancelsOther = 'oco'
+  }
+
+  enum OrderTypes {
+    Limit = 'limit',
+    Market = 'market'
+  }
+
+  enum IntervalTypes {
+    FiveMinute = '5minute',
+    TenMinute = '10minute'
+  }
+
+  enum SpanTypes {
+    Week = 'week',
+    Day = 'day'
+  }
+
+
+  interface RobinhoodWebApi {
+    /**
+     * Revokes the current token for this session.
+     * @param callback
+     */
+    expire_token (callback: request.RequestCallback): void
+
+    /**
+     * Returns the token for the current session.
+     */
+    auth_token (): string|null
+
+    /**
+     * Get the current user's investment profile.
+     * @param callback
+     */
+    investment_profile (callback: request.RequestCallback): void
+
+    /**
+     * Set the MFA code for the user if required.
+     * @param code
+     * @param callback
+     */
+    set_mfa_code (code: string, callback: InitCallback): void
```

Comment:
This is referenced in the docs but is not in the code I see in `master`. Should it be here or removed?

Reply:
We should either remove it from here or add it back in the code if there is still support for it I suppose

Reply:
Gotcha. I can remove for now and can add back if it changes.